### PR TITLE
Add logging functionality for ceph

### PR DESCRIPTION
**Pull request by sivakom**

### CODE REVIEW

**Comment by j2sol** [comment]
LGTM. :+1:

**Comment by j2sol** [comment]
ok to test

File: roles/ceph-common/defaults/main.yml [hunk 1]
```diff
@@ -105,3 +105,25 @@ ceph_common:
...
+    - paths:
+        - /var/log/ceph/ceph-osd*.log
+      fields:
+        type: openstack
+        tags: ceph,ceph-osd
+    - paths:
+        - /var/log/ceph/ceph-audit.log
+      fields:
+        type: openstack
+        tags: ceph,ceph-audit
```
Comment (masteinhauser) [inline roles/ceph-common/defaults/main.yml]: If this should be stored longer term, this should also have its own `audit` tag as Logstash does key off of that in where logs are sent for duplicated storage.

**Comment by j2sol** [comment]
This needs a rebase now that the ports PR has merged.

**Comment by sivakom** [comment]
have to reopen the PR because I ran into some git issues. otherwise, code's been rebased and changes to ceph-audit has been made

**Comment by j2sol** [comment]
ok to test

File: roles/ceph-common/meta/main.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 dependencies:
   - role: monitoring-common
+  - role: logging-config
+  service: ceph
+  logdata: "{{ ceph_common.logs }}"
```
Comment (j2sol) [inline roles/ceph-common/meta/main.yml]: These lost some whitespace. Need to indent them to line up.

**Pull request merged**